### PR TITLE
MTP-1813: Request an intel tool account

### PR DESCRIPTION
**Pull request by xoen**
### What
Users can request an account directly. 

It uses [existing 'Account Request' functionality in `api`](https://github.com/ministryofjustice/money-to-prisoners-api/blob/8f7882917a5ccce1d0bd7a7a529e9bab523590de/mtp_api/apps/mtp_auth/urls.py#L11) with some tweaks.

⚠️ **NOTE**: As prison is optional for these account requests and as a new "Your manager's email" field needs to be added, this PR is [dependent on the corresponding `api` PR](https://github.com/ministryofjustice/money-to-prisoners-api/pull/649) to be merged.

### Links
Ticket: https://dsdmoj.atlassian.net/browse/MTP-1813
Prototype starts: https://pm-intelligence-tool.herokuapp.com/sign-in

### CODE REVIEW

File: mtp_noms_ops/apps/mtp_auth/forms.py
Comment (Staberinde) [inline mtp_noms_ops/apps/mtp_auth/forms.py:42]: So this isn't actually true; it's FIU that review Intelligence Tool requests. But i'm not quite sure why you're adding this in the first place?

File: mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html
Comment (Staberinde) [inline mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html:8]: You can hardcode approver, as it's always going to be FIU

Comment (xoen) [reply mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html:8]: 🤔 It's true actual account request is always approved by FIU but in Justin prototype:
1. First request confirmation message says: ["Once **a member of the Financial Investigations Unit** has approved [...]"](https://pm-intelligence-tool.herokuapp.com/request_access_confirmation)
2. Subsequent request confirmation message says: ["Once **Family Services** has approved [...]"](https://pm-intelligence-tool.herokuapp.com/request_access_already_confirmation)

I'll double-check with Justin 🤷 

PS: This is also why when we submit a ticket to ZenDesk we set the approver to "Family Services".

File: mtp_noms_ops/apps/mtp_auth/forms.py
Comment (ushkarev) [inline mtp_noms_ops/apps/mtp_auth/forms.py:5]: be careful about importing `gettext` vs `gettext_lazy`. it's customary in django to import the lazy version as `_`, and in fact that's the one you want. not using the lazy version means that class-level attributes would be locked to the _first_ language that loads

Comment (xoen) [reply mtp_noms_ops/apps/mtp_auth/forms.py:5]: I see what you mean about the locking of the language 🤔 

Reason why I used the non-lazy version is because I'm using this here to translate a string that goes in the session (`Family Services` as for subsequent requests the message will be slightly different)...it turns out that you can't serialise the output of the lazy version 😬 

I thought that was simple and good enough but maybe we just use a simple constant in the session and make the translation in the view, I'll change 👍

Comment (xoen) [reply mtp_noms_ops/apps/mtp_auth/forms.py:5]: As discussed, removed and simplified this thanks to the fact success message will be always the same.

File: mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html
Comment (xoen) [reply mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html]: FYI: I was following the prototype, but turned out we actually don't want to show a different message so [I simplified the code](https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/531/commits/391efe0259e2c1d7539652fc0c47196b4d098e9d) and can resolve this and the other related comment.

File: mtp_noms_ops/apps/mtp_auth/forms.py
Comment (xoen) [reply mtp_noms_ops/apps/mtp_auth/forms.py]: FYI: [Showing always the same success message now](https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/531/commits/391efe0259e2c1d7539652fc0c47196b4d098e9d) - resolving this.

File: mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html
Comment (ushkarev) [inline mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html:7]: you can drop the `with approver=approver|wrapwithtag:'strong'` now as the param isn't used anymore

Comment (xoen) [reply mtp_noms_ops/templates/mtp_common/user_admin/sign-up-success.html]: I thought I got rid of that but well spotted. Sorted: 168abff82171c5d2fa570a00ec5f0bc48115b4f6

**Review by ushkarev — APPROVED** [review]
looks legit 👍

**Comment by xoen** [comment]
Cheers @ushkarev ☺️

**Pull request merged**
